src/Unused/Types.hs

```diff
@@ -37,20 +43,56 @@ newtype DirectoryPrefix = DirectoryPrefix String deriving (Eq, Show, Ord)
 
 resultsFromMatches :: [TermMatch] -> TermResults
 resultsFromMatches m =
-    TermResults
-        { matches = m
+    calculateLikelihood $ TermResults
+        { resultTerm = resultTerm' terms
```

Comment:
It's common to prefix fields in records like `TermResults` to avoid conflicts between different records or bindings like `resultTerm'`.